frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.scss

```diff
@@ -3,3 +3,7 @@
   padding-right: 30px;
   margin-top: 20px;
 }
+
+button#kubevirt-vm-details__file-systems--info-icon {
```

Comment:
I'd remove this padding and just use the default spacing from PF. It's better accessibility with a larger touch area.

Reply:
If I remove the padding, there is no space between the icon and the header text.

Reply:
If you switch it to a plain modifier and drop the inline variant you should get the proper spacing.

Reply:
Done!